frappe/utils/scheduler.py

```diff
@@ -24,7 +24,7 @@ def start_scheduler():
 	'''Run enqueue_events_for_all_sites every 2 minutes (default).
 	Specify scheduler_interval in seconds in common_site_config.json'''
 
-	schedule.every(60).seconds.do(enqueue_events_for_all_sites)
+	schedule.every(frappe.get_conf().scheduler_tick_interval or 60).seconds.do(enqueue_events_for_all_sites)
```

Comment:
This cannot be greater than `fast_job_interval` or else `fast_job_interval` will be pointless.

For fresh installations where `fast_job_interval` & `scheduler_tick_interval` is not set, each "Fast" jobs will be executed after 1min (but the default interval for "Fast" job is 30secs)

Consider adding a check for this case.

Reply:
Removed `Fast` job types and `fast_job_interval` these can now be used with `cron`

e.g. `* * * * * */5` to schedule a job every 5 seconds. Of course, for this to work as expected `scheduler_tick_interval` must be set appropriately.